### PR TITLE
:wave: openai_transformers

### PR DESCRIPTION
**Pull request by DeNeutoy**
Related to #2270

### CODE REVIEW

File: allennlp/data/tokenizers/pretrained_transformer_pre_tokenizer.py [hunk 1]
```diff
@@ -2,42 +2,10 @@
...
-            for tokens in self.spacy.pipe(standardized_sentences, n_threads=-1)
-        ]
-
-    @overrides
-    def tokenize(self, text: str) -> List[Token]:
-        # This works because our Token class matches spacy's.
-        return _remove_spaces(self.spacy(self._standardize(text)))
 
 
 @Tokenizer.register("bert-basic")
```
Comment (matt-gardner) [inline allennlp/data/tokenizers/pretrained_transformer_pre_tokenizer.py]: We'll want to remove the old BERT ones too, which will let us remove this file entirely.  That can be done as soon as #3634 is merged.

Comment (DeNeutoy) [reply allennlp/data/tokenizers/pretrained_transformer_pre_tokenizer.py]: let's do that in pieces, so we can keep it tracked.

Comment (matt-gardner) [reply allennlp/data/tokenizers/pretrained_transformer_pre_tokenizer.py]: I agree, just noting.

Comment (ZhaofengWu) [reply allennlp/data/tokenizers/pretrained_transformer_pre_tokenizer.py]: I might be missing something but I'm not touching the `pre_tokenizer` class in #3634. So it shouldn't be relevant?

Comment (matt-gardner) [reply allennlp/data/tokenizers/pretrained_transformer_pre_tokenizer.py]: #3634 takes the logic that was in the `BertIndexer` and related classes and implements it for all transformers in the huggingface library.  This makes the `Bert*` classes (including the one mentioned here) obsolete, so they can be removed.

**Comment by DeNeutoy** [comment]
@matt-gardner you used an emoji... Are you ok??

**Pull request merged**